src/constants/other/data-helpers-other.js

```diff
@@ -302,3 +305,132 @@ export function generateSeriesForClock(options) {
      }
   }];
 }
+
+export function generateRandomPointsBoxplot(min, max, numberOfPoints) {
+  const result = [];
+  for (let i = 0; i < numberOfPoints; i += 1) {
+    const rawRandValue = min + (Math.random() * (max - min));
+    const randomValue = Math.round(rawRandValue * 100) / 100;
```

Comment:
In the future all these things should be changed to helper function **limitNumberOfDecimals**